src/features/deletePatient.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Delete Patient
+
+  Background:
+    Given setup logs in the system
+
+  @deletePatient
+  Scenario: Delete Patient
+  And  User clicks on search Patient Record app
+  And  Search for a patient "<patientName>" patientName and load their cover page
+  Then Click 'Delete Patient'
```

Comment:
the last step is 
`Then Patient deleted successfully message appears, and redirected to patient search page`

Reply:
And also ` Then Click 'Delete Patient'`  i think should be `  And Click 'Delete Patient'`

Reply:
@Then annotation is used as an expected output or result.
@And is used to provide logical statements btn two scenarios, so in this case when I use And ,it will not bring it out in a more logical way

Reply:
```
Feature: Delete Patient

  Background:
    Given setup logs in the system

  @deletePatient
  Scenario: Delete Patient
  When  User clicks on search Patient Record app
  And  Search for a patient "<patientName>" patientName and load their cover page
  And Click 'Delete Patient'
  Then Patient deleted successfully message appears, and redirected to patient search page
   Examples:
      | patientName  | REASON |
      | moses mutesa | patient discharged|
```